sass/elements/_latest-posts.scss

```diff
@@ -0,0 +1,53 @@
+.wp-block-latest-posts {
+	@include light-on-dark;
+	font-family: $font-button;
+	font-weight: 700;
+	margin: spacing(2) 0;
+
+	li {
+		background: $grey--alt;
+		color: $white;
+		padding: spacing(2);
+		border-radius: $size__border-radius;
+	}
+
+	a {
+		@include font-size( 2.4 );
+		line-height: $font__line-height--small;
+		color: $white;
+		text-decoration: none;
+
+		&:hover,
+		&:focus,
+		&:visited {
+			color: $white;
+		}
+
+		&:hover,
+		&:focus {
+			text-decoration: underline;
+		}
+	}
+
+	time {
+		color: rgba(255, 255, 255, 0.7);
```

Comment:
This is a nitpick, but can you do `rgba($white, 0.7);` instead? Just to stay consistent with the color vars.

Reply:
Not a problem. :)